test/bridge.jl

```diff
@@ -76,6 +76,26 @@ MOIB.@bridge RSOCtoPSD MOIB.RSOCtoPSDCBridge () () (RotatedSecondOrderCone,) ()
 MOIB.@bridge LogDet MOIB.LogDetBridge () () (LogDetConeTriangle,) () () () (VectorOfVariables,) (VectorAffineFunction,)
 MOIB.@bridge RootDet MOIB.RootDetBridge () () (RootDetConeTriangle,) () () () (VectorOfVariables,) (VectorAffineFunction,)
 
+# Test deletion of bridge
+function test_delete_bridge(m::MOIB.AbstractBridgeOptimizer, ci::MOI.ConstraintIndex{F, S}, nvars::Int, nocs::Tuple) where {F, S}
+    @test MOI.get(m, MOI.NumberOfVariables()) == nvars
+    test_noc(m, F, S, 1)
+    for noc in nocs
+        test_noc(m, noc...)
+    end
+    @test MOI.isvalid(m, ci)
+    @test MOI.candelete(m, ci)
+    MOI.delete!(m, ci)
+    @test !MOI.isvalid(m, ci)
+    @test isempty(m.bridges)
+    test_noc(m, F, S, 0)
+    # As the bridge has been removed, if the constraints it has created where not removed, it wouldn't be there to decrease this counter anymore
+    @test MOI.get(m, MOI.NumberOfVariables()) == nvars
```

Comment:
Do bridges change the value of `MOI.NumberOfVariables()` that's visible to users?

Reply:
If it adds variables as part of the reformulation, it needs to hide it from the user

Reply:
Good, just double checking!